### ISSUE TITLE
IDEAS

### ISSUE THREAD

**Issue by kamilbl** [kamilbl/NuBot#9]

**Comment by tezeug** [comment]
I am currently thinking about the best strategy which could be integrated into the SD.

The strategies could be based e. g. on the avg24h change (like if it is +1.5 % then we can guess it will keep going a little bit up or otherwise) + volume maybe.
It could also work with some normal indicators like StochRSI.

Something like this. I’ll focus on and think of it more today and tomorrow.

**Comment by tezeug** [comment]
In my opinion, the time when to start and stop trading is alfa and omega here in SD. The good thing about it is that it can make nice profits and does not depend on the market situation too much in reality (it can profit when it goes up and down too) but it must know when it most probably should stop and also start trading again.

**Comment by tezeug** [comment]
My another idea also lead to a connection with another exchange, some huge where are also big market makers and where the bot could watch the situation therefore then it could guess what happens on Binance next. E. g. when it would have seen that the price of each pair/coin is starting to go down (based on avg1h change or something like this or big sell walls) here then it could stop trading on Binance in time and could protect against losses.

**Comment by kamilbl** [comment]
> My another idea also lead to a connection with another exchange, some huge where are also big market makers and where the bot could watch the situation therefore then it could guess what happens on Binance next. E. g. when it would have seen that the price of each pair/coin is starting to go down (based on avg1h change or something like this or big sell walls) here then it could stop trading on Binance in time and could protect against losses.

Yes, Yes, Yes... Bot should analiz prices coins in another market...

**Comment by kamilbl** [comment]
I need to think strategy SD. In this strategi is verrry big potential..

**Comment by tezeug** [comment]
Another idea is python TensorFlow but I am not so sure about that, the “strategy” (it might analyze the history of trades / price), also about the impact on the bot and the difficulty of the development. I’m just putting another idea on the table.

**Comment by kamilbl** [comment]
Now I have a lot of work... In head I have a lots ideas... Now I need to finished this strategies...

**Comment by kamilbl** [comment]
Hello. 
What do you think about this... in strategy SD we can use “order take profit”. When depth is >min_depth create order take profit 
It is easy.

**Comment by tezeug** [comment]
Looks good to me.

**Comment by tezeug** [comment]
You could also add something like modes between which you could choose and which would have different setups:

**Comment by tezeug** [comment]
Currently, I have been waiting for some finished strategies and then we can test it, comment and adjust according to the results.

**Comment by kamilbl** [comment]
Updaded. You can test this strategies. Now I will work on strategy SD ... Schemat and update code (Telegram and algoritm).

**Comment by tezeug** [comment]
Do you think it would be possible to integrate some machine learning into SD which would e. g. analyze some history trades later too? Maybe TensorFlow in Python?

**Comment by kamilbl** [comment]
Hmm why not :-) I think it’s possible.

**Comment by kamilbl** [comment]
Please check... [Link](Schemes/Diagram Strategy SDv2.pdf)

**Comment by tezeug** [comment]
I am going to check it right now.

**Comment by tezeug** [comment]
It looks really good. Exactly like in my dreams 😄. Now just to make it real. Maybe that you could add (in the future) also some machine learning (which would exactly also analyse the Binance history data or something like that). But currently I would not change anything on that.

**Comment by kamilbl** [comment]
I found very interesting page [LINK](https://projekter.aau.dk/projekter/files/281297453/On_Machine_Learning_Based_Cryptocurrency_Trading.pdf)
